crates/iota-grpc-api/build.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+fn main() {
+    tonic_build::configure()
+        .protoc_arg("--experimental_allow_proto3_optional")
```

Comment:
remove this flag, it is not experimental since at least 4 years now ;-)
https://github.com/protocolbuffers/protobuf/releases/tag/v3.15.0

Reply:
Thanks for identifying this. Removed this unnecessary flag at [6cfcb25](https://github.com/iotaledger/iota/pull/7511/commits/6cfcb25b899f07957554da3dea0cc5854ddfc6f3).

Reply:
which protobuf version are you using?

Reply:
The version is `libprotoc 29.3`.